### DISCUSSION TITLE
队列和消费组是一对一的关系嘛？

### DISCUSSION THREAD

**Discussion by yfgcq** [apache/rocketmq#7701] [Q&A]
如果队列和消费组一对一关系
rocketmq队列数大于消费组，是不是就存在有队列的消息没被消费？

**Comment by lizhanhui** [comment]
消费组是消费者负载均衡组的意思。消费组订阅(subscribe)一个或者多个主题（topic）, 从而持有若干队列。隶属于同一个消费组的多个消费者根据消费组的属性(attribute: FIFO， standard，transaction isolation等）分配持有的队列的消息。分配方式支持多种，比如standard 消息按照负载分配，顺序消息按照message group展开后分配，流式消费按照队列绑定和分配。 进一步可以看下网站上核心概念的介绍。

**Comment by RongtongJin** [comment]
> 如果队列和消费组一对一关系 rocketmq队列数大于消费组，是不是就存在有队列的消息没被消费？

参考下 https://rocketmq.apache.org/zh/docs/4.x/consumer/01concept2 和 https://rocketmq.apache.org/zh/docs/featureBehavior/08consumerloadbalance
